mobile/scripts/buildStatusGo.sh

```diff
@@ -34,7 +34,7 @@ GOFLAGS="" CGO_ENABLED=1 GOOS="$OS" GOARCH="$GOARCH" \
 	go build \
 		-buildmode="$BUILD_MODE" \
 		-tags 'gowaku_no_rln nowatchdog disable_torrent' \
-		-ldflags="-X github.com/status-im/status-go/vendor/github.com/ethereum/go-ethereum/metrics.EnabledStr=true" \
+		-ldflags="-checklinkname=0 -X github.com/status-im/status-go/vendor/github.com/ethereum/go-ethereum/metrics.EnabledStr=true" \
```

Comment:
Right, I needed to add this on golang 1.24 to be able to build status-desktop on mobile @alexjba :)

Reply:
we have this issue for that task : https://github.com/status-im/status-desktop/issues/18377